src/components/editions/headline.tsx

```diff
@@ -52,8 +52,38 @@ const standardFontStyles = css`
 		${headline.medium({ lineHeight: 'tight' })}
 	}
 `;
+const interviewStyles = css`
+	margin-left: 0.75rem;
+	border: 0;
+`;
+
+const interviewFontStyles = css`
+	${articleWidthStyles}
```

Comment:
We shouldn't need to set the width here as it's already on the `Header` container.

Reply:
Nice spot  - removed.